pages/signup.js

```diff
@@ -109,6 +111,7 @@ const Signup = () => {
             <div className="input-wrapper">
               <label htmlFor="type">Account type</label>
               <select
```

Comment:
I didn't notice before, but your select options should probably be set up in a format that matches the select options api:

```
const accountTypes = [
  { value: "", text: "Choose account type", disabled: true },
  { value: "label", text: "Label" },
  { value: "artist", text: "Artist" },
]
```

You don't need to pass an initial `type` value to the `newUser` object, the select input will default to the first value. And then the select would be:

```
<select
    className="input"
    onChange={handleChange}
    id="type"
    value={newUser.type}
    required
  >
   {accountTypes.map((type) => (
      <option
        key={type.value}
        value={type.value}
        disabled={type.disabled}
      >
        {type.text}
      </option>
    ))}
  </select>
```